### DISCUSSION TITLE
Bug: Suggestion block button missing from toolbar in PR line comments

### DISCUSSION THREAD

**Discussion by alexwforsythe** [community/community#32339] [Pull Requests]
## Expected behavior

When adding a line comment to a PR from the **Files changed** tab, there should be a <kbd>±</kbd> button in the comment form Markdown toolbar that inserts a "suggestion" block containing the contents of the line(s) I'm commenting on.

Source: [Review changes / Comment on a PR / Adding line comments to a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request), step 6

## Actual behavior

There is no <kbd>±</kbd> button in the Markdown toolbar when adding line comments.

Single line:
<img width="846" alt="image" src="https://user-images.githubusercontent.com/1639061/189198289-82617553-bf51-4788-881e-f3b4eb28a210.png">

Multiple lines:
<img width="884" alt="image" src="https://user-images.githubusercontent.com/1639061/189198805-549f6d9d-42e7-45ca-b3e6-96a5fc1d65e6.png">

The button only appears when replying to existing comments:

<img width="886" alt="image" src="https://user-images.githubusercontent.com/1639061/189198478-c89faacd-9035-452f-a014-eb5edf19d6c3.png">

## Extra info

* I'm not using any custom stylesheets or extensions that alter the behavior of GitHub
* The screenshots were taken from interactions on https://github.com/alexwforsythe/code-blocks/pull/185
* I'd also like to request that the button appear when adding line comments from the **Commits** tab, which apparently isn't supported according to #7128

**Comment by svenja-haas** [comment]
<img width="826" height="252" alt="grafik" src="https://github.com/user-attachments/assets/3e4c5303-4fee-493c-9c48-46055301c9f5" /> <br>
This button is also nonexistent for me in the "files changed" tab. No idea why.

Edit: Sooo it was because the PR was merged while I was reviewing, so I didn't notice. Would prefer it to be greyed out with a tooltip for more clarity in this case.
